### DISCUSSION TITLE
Can custom `.me` domain registration get extended automatically through renewing github student pack?

### DISCUSSION THREAD

**Discussion by edgexyz** [community/community#17703] [GitHub Education]
I registered one domain about half a year ago, and I'm going to renew my github student recently. I'd like to know whether I can use the renewed 1 year free domain provided by the pack to extend my existing domain registration.

**Comment by vincenttjia** [comment]
I believe no, since it's a one time offer, extending your pack won't extend your domain validity.

But I believe you can claim other domain such as .tech and name.com(Only do this once your .me domain is nearing it's expiry date or already expired since they would only last 1 year from registration)
